libs/dh/market-participant/actors/feature-organizations/src/lib/table/dh-table.component.html

```diff
@@ -26,6 +26,7 @@
     [loading]="isLoading"
     sortDirection="asc"
     [sortClear]="false"
+    (rowClick)="onRowClick($event)"
```

Comment:
suggestion: to highlight the selected row.

```suggestion
    [activeRow]="activeRow"
    (rowClick)="onRowClick($event)"
```